mite_cli.py

```diff
@@ -21,6 +21,14 @@
 
 
 def parse_date(inp):
+    day = timedelta(days=1)
```

Comment:
Maybe we should use something like [dateparser](https://github.com/scrapinghub/dateparser) here instead of rolling it ourselves?

Reply:
That makes sense, I will switch it out.